spec/components/metrics_summary_component/view_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   it "renders the start and end dates" do
-    expect(page).to have_css("h2", text: metrics_summary.formatted_date_range)
+    expect(page).to have_css("h2", text: Nokogiri::HTML(metrics_summary.formatted_date_range).text)
```

Comment:
I missed this in the other PR but saw your discussion with Al. 

Could we replace this with?:
```
expect(page).to have_css("p", text: "Sorry, there's a problem getting your form's metrics.")
expect(page).to have_css("p", text: "Try again later.")
```

Reply:
I can do this, but the downside is our tests will break if we change the content. Been playing around with it and found that I can do this, which I think might be less fragile:
```suggestion
    expect(render_inline(metrics_summary).to_html).to include(metrics_summary.formatted_date_range)
```
and in the error cases that would look like: 
```
      expect(render_inline(metrics_summary).to_html).to include(metrics_summary.error_message)
```

Reply:
I've updated to do the above @chao-xian

Reply:
Nice! 

(As an aside: I'm personally not against tests breaking from content changes as to me being that explicitness and better readability is easier to parse but I'm not going to start that crusade)

Reply:
Definitely worth having the conversation at some point, because there are ways it would make life easier